cmd/minikube/cmd/start.go

```diff
@@ -737,6 +737,10 @@ func suggestMemoryAllocation(sysLimit int, containerLimit int) int {
 	// Suggest 25% of RAM, rounded to nearest 100MB. Hyper-V requires an even number!
 	suggested := int(float32(sysLimit)/400.0) * 100
 
+	if nodes := viper.GetInt(nodes); nodes > 1 {
+		suggested /= nodes
```

Comment:
this needs examples, or unit test. for example what would be the susggeted for 2 nodes on a laptop with 8 gb, or 16 gb, or 3 nodes on ...

Reply:
the fallback is still 2200 MB, since minikube nodes don't do very well with less then 2 GB of RAM. I'm literally taking what we already do when dividing by the number of nodes we're expecting.

Reply:
unit tests added